install.sh

```diff
@@ -115,7 +115,11 @@ if [ "$1" = "install" ] || [ "$1" = "config" ]; then
     sudo apt install git build-essential kernel-package fakeroot libncurses5-dev libssl-dev ccache bison flex qtbase5-dev ${clang_deps} -y
   elif [ "$_distro" = "Fedora" ]; then
     msg2 "Installing dependencies"
-    sudo dnf install fedpkg fedora-packager rpmdevtools ncurses-devel pesign grubby qt5-devel libXi-devel gcc-c++ git ccache flex bison elfutils-libelf-devel openssl-devel dwarves rpm-build ${clang_deps} -y
+    if [ "rpm -E %fedora" = "32" ]; then
```

Comment:
This wont work, you should try the changes! :D 

Maybe be something like:

```shell
if [ $(rpm -E %fedora) = "32" ]; then
```

Reply:
still got the same error here

erro: Status de saída de /var/tmp/rpm-tmp.GkSCMD inválido (%build)


Erros na construção do RPM:
    line 20: It's not recommended to have unversioned Obsoletes: Obsoletes: kernel-headers
    Status de saída de /var/tmp/rpm-tmp.GkSCMD inválido (%build)
make[1]: *** [scripts/Makefile.package:59: rpm-pkg] Erro 1
make: *** [Makefile:1523: rpm-pkg] Erro 2
 -> exit cleanup done

elif [ "rpm -E %fedora" = "32" ]; then
    msg2 "Installing dependencies"
    sudo dnf install fedpkg fedora-packager rpmdevtools ncurses-devel pesign grubby qt5-devel libXi-devel gcc-c++ git ccache flex bison elfutils-libelf-devel openssl-devel dwarves rpm-build ${clang_deps} -y

Reply:
It is outdated try the newest push.

Reply:
> It is outdated try the newest push.

its the same error here, theres some trick to do with F33?
to pass from dependencies I need to change "qt5-devel to qt-devel" and still got same error of kernel-headers above,
since installation of F33 I can't compile kernel and wine-tkg, I thinking go back to F32 for a wile....

Reply:
I am using Fedora 33, you need to install more dependencies.
`sudo dnf install fedpkg fedora-packager rpmdevtools ncurses-devel pesign grubby qt5-devel libXi-devel gcc-c++ git ccache flex bison elfutils-libelf-devel openssl-devel dwarves rpm-build ${clang_deps} -y`

Reply:
How do you install qt5-devel, its not avaiable on fedora/rpmfusion repo.... in my machine

Reply:
qt5-devel ins only avaiable for F32, I downloaded this file and install here on F33, still not working, I'll apreciate some help to this....

Reply:
qt5-devel is not in Fedora 33, it is installed by default I believe if not - try to do qt5-base-devel.

Reply:
> qt5-devel is not in Fedora 33, it is installed by default I believe if not - try to do qt5-base-devel.

Hi,

I can satisfied dependencies with "qt5-qtbase-devel.x86_64 qt5-qtbase-devel.i686" packges, but I still get the same error with Obsoletes: kernel-headers, man I realy want to give up, with this.....
I'm using stock kernel, with nothing, maybe my nvidia driver can doing something, I really have no ideia what happen....

Reply:
Yes, Nvidia driver does not work with Kernel 5.9. :(

Reply:
I know about nvidia problems with kernel 5.9, but the problem here it's not compiling kernel, 5.8 or 5.9, just because script don't finish compilation

Reply:
You need these packages:
`Package bash-5.0.17-2.fc33.x86_64 is already installed.
Package bc-1.07.1-11.fc33.x86_64 is already installed.
Package binutils-2.35-14.fc33.x86_64 is already installed.
Package bison-3.6.4-3.fc33.x86_64 is already installed.
Package bzip2-1.0.8-4.fc33.x86_64 is already installed.
Package diffutils-3.7-7.fc33.x86_64 is already installed.
Package dwarves-1.17-4.fc33.x86_64 is already installed.
Package elfutils-0.182-1.fc33.x86_64 is already installed.
Package elfutils-devel-0.182-1.fc33.x86_64 is already installed.
Package findutils-1:4.7.0-7.fc33.x86_64 is already installed.
Package flex-2.6.4-5.fc33.x86_64 is already installed.
Package gawk-5.1.0-2.fc33.x86_64 is already installed.
Package gcc-10.2.1-6.fc33.x86_64 is already installed.
Package git-core-2.28.0-1.fc33.x86_64 is already installed.
Package gzip-1.10-3.fc33.x86_64 is already installed.
Package libkcapi-hmaccalc-1.2.0-3.fc33.x86_64 is already installed.
Package hostname-3.23-3.fc33.x86_64 is already installed.
Package kmod-27-3.fc33.x86_64 is already installed.
Package m4-1.4.18-15.fc33.x86_64 is already installed.
Package make-1:4.3-2.fc33.x86_64 is already installed.
Package net-tools-2.0-0.58.20160912git.fc33.x86_64 is already installed.
Package nss-tools-3.57.0-1.fc33.x86_64 is already installed.
Package openssl-1:1.1.1g-15.fc33.x86_64 is already installed.
Package openssl-devel-1:1.1.1g-15.fc33.x86_64 is already installed.
Package patch-2.7.6-13.fc33.x86_64 is already installed.
Package perl-Carp-1.50-457.fc33.noarch is already installed.
Package perl-devel-4:5.32.0-463.fc33.x86_64 is already installed.
Package perl-generators-1.11-9.fc33.noarch is already installed.
Package perl-interpreter-4:5.32.0-463.fc33.x86_64 is already installed.
Package pesign-113-13.fc33.x86_64 is already installed.
Package python3-devel-3.9.0-1.fc33.x86_64 is already installed.
Package redhat-rpm-config-172-1.fc33.noarch is already installed.
Package rpm-build-4.16.0-1.fc33.x86_64 is already installed.
Package tar-2:1.32-6.fc33.x86_64 is already installed.
Package xz-5.2.5-3.fc33.x86_64 is already installed.
`

Reply:
> kmod

"perl-devel perl-generators" was missing here but not solve my problem, how can I make a full log to paste here, theres no .log files here.

Reply:
Drag a text file toward the bottom of the textbox (attach files).

Reply:
> You need these packages:
> `kmod, patch, bash, tar, git-core bzip2, xz, findutils, gzip, m4, perl-interpreter, perl-Carp, perl-devel, perl-generators, make, diffutils, gawk gcc, binutils, redhat-rpm-config, hmaccalc, bison, flex`
> 
> Let me know which one you are missing. I can include them in the install script if they fix your issue.

Looks like was working now, I can compile both kernels 5.8/5.9 with no errors, acording your tips I installed this packages here, 3 was missing, this was installed here to solve....

 bash-5.0.17-2.fc33.x86_64 bc-1.07.1-11.fc33.x86_64 binutils-2.35-14.fc33.x86_64   bison-3.6.4-3.fc33.x86_64   bzip2-1.0.8-4.fc33.x86_64   diffutils-3.7-7.fc33.x86_64   dwarves-1.17-4.fc33.x86_64   elfutils-0.182-1.fc33.x86_64   elfutils-devel-0.182-1.fc33.x86_64   findutils-1:4.7.0-7.fc33.x86_64   flex-2.6.4-5.fc33.x86_64   gawk-5.1.0-2.fc33.x86_64   gcc-10.2.1-6.fc33.x86_64   git-core-2.28.0-1.fc33.x86_64   gzip-1.10-3.fc33.x86_64   libkcapi-hmaccalc-1.2.0-3.fc33.x86_64   hostname-3.23-3.fc33.x86_64   kmod-27-3.fc33.x86_64   m4-1.4.18-15.fc33.x86_64   make-1:4.3-2.fc33.x86_64   net-tools-2.0-0.58.20160912git.fc33.x86_64   nss-tools-3.57.0-1.fc33.x86_64   openssl-1:1.1.1g-15.fc33.x86_64   openssl-devel-1:1.1.1g-15.fc33.x86_64   patch-2.7.6-13.fc33.x86_64   perl-Carp-1.50-457.fc33.noarch   perl-devel-4:5.32.0-463.fc33.x86_64   perl-generators-1.11-9.fc33.noarch   perl-interpreter-4:5.32.0-463.fc33.x86_64   pesign-113-13.fc33.x86_64   python3-devel-3.9.0-1.fc33.x86_64   redhat-rpm-config-172-1.fc33.noarch   rpm-build-4.16.0-1.fc33.x86_64   tar-2:1.32-6.fc33.x86_64   xz-5.2.5-3.fc33.x86_64 qt5-qtbase-devel.x86_64 qt5-qtbase-devel.i686 perl-devel perl-generators openssl,

Now I'll install here, lets see wats happen....

Reply:
I will just include them all in the requirements.

Reply:
> I will just include them all in the requirements.

Nice!

Just some observation, I'm reproducing now in my old laptop this compilation with kernel 5.8 and Fedora 33, I just able to pass from dependencies when I installed "qt5-devel-5.13.2-2.fc32.noarch.rpm Fedora 32 for x86_64  " downloaded from rpmfind, once this package its not avaiable from Fedora 33 yet, maybe some people i'll still have this problem, tks for your help and patience, now I'm moving to wine-tkg thats not correctly installing on Fedora 33, if you have some time please help there too....Nice!

Reply:
Wine-TKG is not worth using unless you need to do specific patches. It's better to use just vanilla Wine-Staging or Proton anyway. If you must use Wine-TKG you should just use the prebuilt ones.

Reply:
> Wine-TKG is not worth using unless you need to do specific patches. It's better to use just vanilla Wine-Staging or Proton anyway. If you must use Wine-TKG you should just use the prebuilt ones.

thats bad, but ok, better to use proton, tks....

Reply:
I would disagree. Wine/proton-tkg prebuilts are the worthless ones. I only offer them because people requested some again and again, but they aren't making much sense IMO. The idea is to be able to build bleeding edge builds with more features than available upstream. If you can't select which features you want/don't want, and can't have daily builds to test latest devel changes, it makes it worthless.
Anyway @tuxtergames feel free to open an issue about your installation problem :wink:

Reply:
@Tk-Glitch I was unable to get it to work on Fedora even when I built it, but it seems to have a lot of setup you have to do to get it working. In Fedora, building DXVK is very difficult a the packages do not work out-of-the-box.

Reply:
When I added initial Fedora (32) support, it was working just fine :frog: I haven't done any test since on that distro, to be fair.

Reply:
Fedora 32 support did not work either.

Reply:
I guess I'll have to reinstall it in a VM for a check. I really am not looking forward to it :disappointed:

Reply:
> Fedora 32 support did not work either.

Actually last sunday before install fedora 33 was working on 32, I was built wine-tkg 20

Reply:
> I would disagree. Wine/proton-tkg prebuilts are the worthless ones. I only offer them because people requested some again and again, but they aren't making much sense IMO. The idea is to be able to build bleeding edge builds with more features than available upstream. If you can't select which features you want/don't want, and can't have daily builds to test latest devel changes, it makes it worthless.
> Anyway @tuxtergames feel free to open an issue about your installation problem 😉

There's one open last week, I comparing with manjaro compilation, fedora finish compilation, but not appear wine folder compilation "non-makepkg-builds" folder, any help will very appreciated there.....

Reply:
Fedora 32 compiles wine-tkg, it just is doesn't run in Lutris or in Wine when I attempt to do so. DXVK also compilation is broken with Fedora in general. I have made a ticket to Redhat to make their packages for compiling Windows .DLL in Fedora better. The packages for it do not really come prepackaged properly.

Reply:
> Fedora 32 compiles wine-tkg, it just is doesn't run in Lutris or in Wine when I attempt to do so. DXVK also compilation is broken with Fedora in general. I have made a ticket to Redhat to make their packages for compiling Windows .DLL in Fedora better. The packages for it do not really come prepackaged properly.

Man I tell you that worked, since 5.17 I'm using here in lutris on fedora 32, I can't install, I just put a symbolic link of bin folder inside .local/, and wine-tkg works like a system wine, I do the same in fedora 33.